picard/tagger.py

```diff
@@ -360,9 +445,106 @@ def load_to_picard(self, items):
             for item in parsed_items.mbids | parsed_items.urls:
                 thread.to_main(file_lookup.mbid_lookup, item, None, None, False)
 
-        if parsed_items:
+        for command in parsed_items.commands:
+            self.handle_command(command)
+
+        if parsed_items.non_executable_items():
             self.bring_tagger_front()
 
+    def get_album_pane_tracks(self):
+        for album in self.albums.values():
+            for track in album.iterfiles():
+                yield track
+
+    def get_all_file_objects(self):
```

Comment:
Can we maybe name it `iter_all_files`? Currently this is easily confused with the similar named `get_files_from_objects` (which we probably should remove anyway, but still)